quickwit/quickwit-indexing/src/actors/packager.rs

```diff
@@ -377,12 +377,7 @@ mod tests {
             }
         }
         index_writer.commit()?;
-        let pipeline_id = IndexingPipelineId {
-            index_id: "test-index".to_string(),
-            source_id: "test-source".to_string(),
-            node_id: "test-node".to_string(),
-            pipeline_ord: 0,
-        };
+        let pipeline_id = MergePipelineId::new("test-index", "test-source", "test-node");
```

Comment:
I'd rather keep `IndexingPipeline` for actors belonging to the indexing chain where there are potentially multiple instances of the actor for the same `(index_id, source_id, node_id)`.

Reply:
Not done yet, I am still thinking about the change I did in `SplitAttrs` because there are some actors that are used both in indexing and merging.